### PR TITLE
Stac 20940 document traces

### PR DESCRIPTION
**Pull request by fvlankvelt**

### CODE REVIEW

File: configure/logging/kubernetes-logs.md [hunk 1]
```diff
@@ -19,8 +19,7 @@ Note that logs stored on pods will be regularly removed. For long term access to
 | StackState function | Logs on pod                                                                                                                                         |
 | :--- |:----------------------------------------------------------------------------------------------------------------------------------------------------|
 | API \(including topology, charts and settings\) | `stackstate-api`                                                                                                                                    |
-| Checks | `stackstate-checks`                                                                                                                                 |
```
Comment (rb3ckers) [inline configure/logging/kubernetes-logs.md:22]: Why drop stackstate-checks from the docs? It is still there but runs the monitors.

File: use/views/k8s-traces-perspective.md [hunk 2]
```diff
@@ -4,55 +4,44 @@ description: StackState for Kubernetes troubleshooting
 
 # Traces perspective
 
-The Traces Perspective shows a list of traces and their spans for the components in your view. This allows you to monitor the performance of the applications in your IT infrastructure directly in StackState.
+The Traces Perspective shows spans for a component and their duration distribution with status code.
```
Comment (rb3ckers) [inline use/views/k8s-traces-perspective.md]: I would drop the `status code` addition in this sentence. It makes the sentence weird: does it refer to distribution or to spans for example? We also don't mention service name, span name or start time.

We could also go for the final situation already and say `spans with their related metrics`. Currently that's only the latency distribution but later also error ratio etc.

File: use/views/k8s-traces-perspective.md [hunk 3]
```diff
@@ -4,55 +4,44 @@ description: StackState for Kubernetes troubleshooting
...
+This allows you to monitor the performance of the applications in your IT infrastructure directly in StackState.
 
-Click on any trace in the list to see the spans that belong to it. Span types are colored differently according to the information on the right. When inspecting a trace and seeing the list of its spans, you can click on any span to see further details. The image below illustrates this action.
+Click on any span in the list to see the descendant spans that belong to it. 
+When expanded, the timeline for a span shows when each descendant started and completed.
+Parent spans in the same trace are collapsed by default, but can be expanced as needed.
```
Comment (rb3ckers) [inline use/views/k8s-traces-perspective.md]: Typo: `expanced`

File: use/views/k8s-traces-perspective.md [hunk 4]
```diff
@@ -4,55 +4,44 @@ description: StackState for Kubernetes troubleshooting
...
-The [topology filters](k8s-filters.md#filter-topology) define the elements \(components and relations\) for which traces will be displayed. Only traces relating to elements that match the applied topology filters, or the view itself, will be visible.
+In addition to these filters, the traces match the **Time Window** selected in the timeline control at the bottom of the StackState UI.
+Adjust the time window to show only traces from that time.
 
-The [trace filters](k8s-filters.md#filter-traces) allow you to refine the traces displayed based on span type and tags.
+## Span details
 
-In addition to these filters, the traces perspective shows traces matching the **Time Window** selected in the timeline control at the bottom of the StackState UI. Adjust the time window to show only traces from that time.
+In StackState, a [view](k8s-view-structure.md) shows you a sub-selection of your IT infrastructure for a particular Kubernetes resource.
+The traces perspective shows the spans related to the resource, along with their descendants and (on request) parents.
```
Comment (rb3ckers) [inline use/views/k8s-traces-perspective.md]: Technically that is not correct. The "Show parent spans" link reveals the entire trace, including the siblings of this span (and even the siblings children etc.). So this is also incorrectly labeled in the StackState UI.

File: use/views/k8s-traces-perspective.md [hunk 5]
```diff
@@ -4,55 +4,44 @@ description: StackState for Kubernetes troubleshooting
...
+The Traces Perspective shows spans for a component and their duration distribution with status code.
+This allows you to monitor the performance of the applications in your IT infrastructure directly in StackState.
 
-Click on any trace in the list to see the spans that belong to it. Span types are colored differently according to the information on the right. When inspecting a trace and seeing the list of its spans, you can click on any span to see further details. The image below illustrates this action.
+Click on any span in the list to see the descendant spans that belong to it. 
+When expanded, the timeline for a span shows when each descendant started and completed.
+Parent spans in the same trace are collapsed by default, but can be expanced as needed.  
+Spans are colored differently according to their OpenTelemetry ServiceName.
+When inspecting a trace and seeing the list of its spans, you can click on any span to see further details.
+The image below illustrates this action.
```
Comment (rb3ckers) [inline use/views/k8s-traces-perspective.md]: The image below actually doesn't illustrate that, it is the other image further down on the page.

File: use/views/k8s-traces-perspective.md [hunk 6]
```diff
@@ -4,55 +4,44 @@ description: StackState for Kubernetes troubleshooting
...
-
-The two images above illustrate these concepts by showing a library application whose main responsibility is to fetch a list of books. You can see an example of a trace and its spans for a request to fetch the list of books and the resulting topology created out of it.
-
-When ingesting traces, StackState attaches service identifiers to the components that are created. These identifiers are also included as part of the **service** property of the spans in a trace. All topology created out of a trace will have a tag **has\_traces**, this allows you to easily identify components for which you have traces.
-
-## Sorting and limits
-
-Traces are sorted by latency \(descending\). This is the only sorting criteria available in this version. The trace list isn't limited by size, you can scroll infinitely to see all traces that are available for your component, filter and time selections. Finally, on top of the list of traces, StackState displays an approximation of the total amount of traces that are returned from the filters you have selected.
+As shown in the second screenshot, it is possible to select a duration interval for the span Duration by brushing the histogram.
```
Comment (rb3ckers) [inline use/views/k8s-traces-perspective.md]: I wouldn't call this zooming in and out but include this in the `Filter traces` section.

File: use/views/k8s-traces-perspective.md [hunk 7]
```diff
@@ -4,55 +4,44 @@ description: StackState for Kubernetes troubleshooting
...
-
-When using the Traces Perspective in live mode, you are constantly polling for the slowest traces in your time range selection. However, in a large IT infrastructure with constant requests being traced, your slowest traces right now might not be your slowest traces in a matter of seconds, changing their position the list. These constant updates to the order of the list could become frustrating, for example, if you are inspecting a trace/span, or scrolling through the list to look for a specific trace or pattern.
-
-To avoid this, time will effectively be paused when you inspect a trace/span or scroll through the list of traces in live mode. This allows you to browse through a stable snapshot of your data. Note that pausing time means that you are now in the past, click **Go live** or **BACK TO LIVE** to stop time travelling and return to live mode.
-
-➡️ [Learn more about time travel](../timeline-time-travel.md#time-travel)
\ No newline at end of file
+When using the Traces Perspective, just like in other perspectives, you can either be in live mode or [time travel to the past](../stackstate-ui/k8sTs-timeline-time-travel.md#time-travel).
```
Comment (rb3ckers) [inline use/views/k8s-traces-perspective.md:48]: Should we mention that, when scrolling through the list, you automatically switch to time travel mode?

File: use/views/k8s-traces-perspective.md (hunk 4 above)
Comment (rb3ckers) [reply use/views/k8s-traces-perspective.md]: Note that this is being changed in the UI now, so the text/screenshot needs an update too.

File: configure/logging/kubernetes-logs.md (hunk 1 above)
Comment (fvlankvelt) [reply configure/logging/kubernetes-logs.md:22]: the Monitors are listed separately a few lines below

File: use/views/k8s-traces-perspective.md (hunk 4 above)
Comment (fvlankvelt) [reply use/views/k8s-traces-perspective.md]: I got the impression it needs to go live NOW, so we can update the screenshot later

File: use/views/k8s-traces-perspective.md (hunk 5 above)
Comment (fvlankvelt) [reply use/views/k8s-traces-perspective.md]: yeah, I think it's pretty ambiguous wording - I'll remove it.  Actually illustrating an action would mean an animated gif with a clear "click".

File: use/views/k8s-traces-perspective.md (hunk 6 above)
Comment (fvlankvelt) [reply use/views/k8s-traces-perspective.md]: I agree - and so does @oboc-sts

File: use/views/k8s-traces-perspective.md (hunk 7 above)
Comment (fvlankvelt) [reply use/views/k8s-traces-perspective.md:48]: not sure; TBH I think that's not really a feature but more a bug that we should address in due time.

**Review by rb3ckers — APPROVED** [review]
LGTM

**Pull request merged**